Makefile

```diff
@@ -173,12 +173,11 @@ $(DOCDIR):
 gendoc: $(DOCDIR) prefixmaps
 	# Copy all documentation files to the documentation directory
 	cp -rf $(SRC)/docs/* $(DOCDIR)
-	# Added copying of images and renaming of TEMP.md
-	cp $(SRC)/docs/*md $(DOCDIR)
-	cp -r $(SRC)/docs/images $(DOCDIR)
```

Comment:
the removed lines we just redundant with `cp -rf $(SRC)/docs/* $(DOCDIR)`, right?